source/helpers/hotfix.ts

```diff
@@ -15,12 +28,17 @@ export const updateHotfixes = cache.function(async (): Promise<string[][]> => {
 		return [];
 	}
 
-	return atob(content)
-		.trim()
-		.split('\n')
-		.map(line => line.split(','));
+	const storage: HotfixStorage = [];
+	for (const [featureID, unaffectedVersion, relatedIssue] of parseCsv(atob(content))) {
+		if (featureID && relatedIssue && (!unaffectedVersion || compareVersions(unaffectedVersion, version) > 0)) {
+			storage.push([featureID as FeatureID, relatedIssue]);
+		}
+	}
+
+	return storage;
 }, {
 	maxAge: {hours: 6},
+	staleWhileRevalidate: {days: 3},
```

Comment:
Hopefully this fixes the situation described in https://github.com/refined-github/refined-github/issues/4861#issuecomment-933148944.

Reply:
@fregante :rocket: ?